models/migrations/migrations.go

```diff
@@ -296,6 +296,8 @@ var migrations = []Migration{
 	NewMigration("Fix migrated repositories' git service type", fixMigratedRepositoryServiceType),
 	// v120 -> v121
 	NewMigration("Add owner_name on table repository", addOwnerNameOnRepository),
+	// v120 -> v121
```

Comment:
```suggestion
	// v121 -> v122
```

(There will be naming conflicts when this is merged, anyway)

Reply:
added to my pull

Reply:
yes He Has to resolve confilcts ...